Octokit.Tests.Integration/Reactive/ObservableEventsClientTests.cs

```diff
@@ -617,7 +618,82 @@ public TheGetAllUserPerformedPublicMethod()
 
         public class TheGetAllForAnOrganizationMethod
         {
-           
+            readonly ObservableEventsClient _eventsClient;
+            readonly string _organization;
+            readonly string _user;
+
+            public TheGetAllForAnOrganizationMethod()
+            {
+                var github = Helper.GetAuthenticatedClient();
+                _eventsClient = new ObservableEventsClient(github);
+                _user = github.User.Current().Result.Login;
+                _organization = github.Organization.GetAllForCurrent().Result.First().Login;
+            }
+
```

Comment:
@shiftkey pls, i need some help with the GetAllForAnOrganization Method's IntegrationTests  ! since the user must be authenticated and the organisation must be associated with that particular user !